prebuild_binaries/linux/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:10-jessie
```

Comment:
If we pre-build on jessie, will the prebuilt binaries work fine on older distributions like some Red Hat or something? Is libstdc++ linked in dynamically or statically for the Linux prebuilt binaries?

Reply:
I used a jessie image because that's what gRPC uses. I assume this should work on other linux distributions -- but not sure. 

I think that libstdc++ would be dynamically linked here right now, but not really familiar yet with  (gRPC has some targets in binding.gyp that have "'type': 'static_library'", but beyond that I haven't looked at this)